pubsublite/internal/wire/subscriber.go

```diff
@@ -537,6 +560,15 @@ func (as *assigningSubscriber) Terminate() {
 	}
 }
 
+// PartitionActive returns whether the partition is still active.
```

Comment:
Have you considered returning a slice of active partitions instead? It's almost as simple for clients, and provides a superset of functionality.

Reply:
Yes, I almost did that, but it didn't work well with the existing data structures of `assigningSubscriber` and `multiPartitionSubscriber`, and so thought it wasn't worth implementing the superset of functionality that isn't required yet. Nothing outside this library can access anything in `internal/`, so we can easily change the interface later.